src/utils/types.ts

```diff
@@ -62,3 +62,45 @@ export interface Request extends IncomingMessage {
 }
 
 export type Rewriter = (req: Request) => Match | null
+
+export interface Account {
+  id: string
+  name: string
+  slug: string
+  type: string
+  capabilities: {
+    sites: {
+      included: number
+      used: number
+    }
+    collaborators: {
+      included: number
+      used: number
+    }
+  }
+  billing_name: string
+  billing_email: string
+  billing_details: string
+  billing_period: string
+  payment_method_id: string
+  type_name: string
+  type_id: string
+  owner_ids: string[]
+  roles_allowed: string[]
+  created_at: string
+  updated_at: string
+}
+
+export interface RepoFromGithub {
```

Comment:
This is more straightforward for a typename
```suggestion
export interface GithubRepo {
```

Reply:
Changed 👍